README.md

```diff
@@ -24,40 +24,46 @@ Please also check out our [Code of Conduct](CODE_OF_CONDUCT.md)!
 
 1.  Install odoo v11. We created [a Boxwise dev-setup](https://github.com/boxwise/boxwise-doodba). It is based on a docker environment for development and production of odoo by Tecnativa. It should just be 5 commands and you are ready to go. You have to have [docker](https://docs.docker.com/install/) and docker-compose installed for this dev-setup.
 
-2.  Clone this folder into the [private src folder](https://github.com/boxwise/boxwise-doodba/tree/11.0_wms/odoo/custom/src/privates) of your setup. **Please do NOT rename the boxwise_wms repository!**
-
-    `git clone https://github.com/boxwise/boxwise_wms.git`
-
-3.  We recommend that you create your own database before you log in. There you can choose your own log in credentials and leave out the demo data of the default installation. 
+2.  We recommend that you create your own database before you log in. There you can choose your own log in credentials and leave out the demo data of the default installation.
 
     - Open up odoo, e.g. `http://localhost:11069/web/database/manager`
     - Click "Create Database"
 
-4. Activate the developer mode! There two ways:
+3. Activate the developer mode! There two ways:
 
     1. Add `debug` in the url --> `localhost:11069/web?debug#....` or
-    2. Go to settings and click the link hidden beneath the credentials on the far right! 
+    2. Go to settings and click the link hidden beneath the credentials on the far right!
 
-5.  Install the `boxwise_wms` module!
+4.  Install the `boxwise_wms` module!
```

Comment:
Can doodba do this automatically for us too??

Reply:
I'll check that out. I imagine it might be possible.